Kitodo/src/main/java/org/kitodo/config/Parameters.java

```diff
@@ -0,0 +1,688 @@
+/*
+ * (c) Kitodo. Key to digital objects e. V. <contact@kitodo.org>
+ *
+ * This file is part of the Kitodo project.
+ *
+ * It is licensed under GNU General Public License version 3 or later.
+ *
+ * For the full copyright and license information, please read the
+ * GPL3-License.txt file that was distributed with this source code.
+ */
+
+package org.kitodo.config;
+
+/**
+ * These constants define configuration parameters usable in the configuration
+ * file. This file reflects the order of the global
+ * {@code kitodo_config.properties} file (as far as possible), and vice versa.
+ */
+public class Parameters {
+
+    /*
+     * =========================================================================
+     * ==== FILE AND DIRECTORY MANAGEMENT
+     * =========================================================================
+     * ==== ----------------------------------- Directories
+     * -----------------------------------
+     */
+
+    // DIR_XML_CONFIG = "KonfigurationVerzeichnis" → org.kitodo.config.Config
```

Comment:
Why did you add this out commented line? If this line is not needed, then do not add them or add at least an explanation to it.